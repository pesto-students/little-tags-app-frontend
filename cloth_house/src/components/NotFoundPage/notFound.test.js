import createComponentWithIntl from '../../utils/helper_snapshot'
import NotFound from "./index"

it('Not Found Snapshot', () => {
    const component = createComponentWithIntl(<NotFound />);
    let tree = component.toJSON()
  
    expect(tree).toMatchSnapshot()
  
  });