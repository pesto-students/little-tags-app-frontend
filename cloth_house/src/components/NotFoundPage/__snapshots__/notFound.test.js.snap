// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Snapshot 1`] = `
<div>
  This Route Doesn't Exist
</div>
`;
